examples/kind/kind-with-registry-non-secure.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
what are the scripts for? If they can be used by users would be good to describe what they are doing in the README.md